src/core/include/openvino/op/util/multi_subgraph_base.hpp

```diff
@@ -201,14 +201,14 @@ class OPENVINO_API MultiSubGraphOp : public Op {
     ///
     /// \param     index sub-graph's index in op
     /// \return pointer to Model with sub-graph
-    virtual const std::shared_ptr<Model>& get_function(int index) const {
+    virtual const std::shared_ptr<Model>& get_function(size_t index) const {
```

Comment:
This could be considered as a change of the public API. I think we need to keep the `int` and add some assertion in the body (+ a `static_cast`)

Reply:
Right, reverted.